### PR TITLE
BEIS ingress v3 - Fuel

### PR DESCRIPTION
**Pull request by maryarbab**
**Is this a new addition or an update to an existing factor?**

- [*] Update

**What category or categories does it affect?**

_Fuel_

**What?**

_Complete BEIS fuel category emissions factors_

**Why?**

_To update the existing EFs and add any missing one from the source._

### CODE REVIEW

File: OpenEmissionFactorsDB.csv [hunk 1]
```diff
@@ -1210,16 +1206,36 @@ Energy,Fuel,fuel_type_aviation_gasoline-fuel_use_aviation,Aviation gasoline,EUR,
...
+Energy,Fuel,fuel_type_avgas-fuel_use_aviation,Aviation spirit,L,,2.33048,2.28297,0.02576,0.02175,,,,1,fuel_combustion,BEIS,2021,2021,2021,GB,,2022/03/20,Emission intensity of stationary combustion of fuel. Fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Energy,Fuel,fuel_type_avgas-fuel_use_aviation,Aviation spirit,tonne,,3192.76,3127.67,35.29,29.8,,,,1,fuel_combustion,BEIS,2021,2021,2021,GB,,2022/3/20,Emission intensity of stationary combustion of fuel. Fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1212]: Think these dates should be "2022/03/20" to align with format elsewhere in the DB?

File: OpenEmissionFactorsDB.csv [hunk 2]
```diff
@@ -1210,16 +1206,36 @@ Energy,Fuel,fuel_type_aviation_gasoline-fuel_use_aviation,Aviation gasoline,EUR,
...
+Energy,Fuel,fuel_type_biopetrol-fuel_use_na,Bio Petrol,L,,0.01402,,,,,,,1,fuel_combustion,BEIS,2021,2021,2021,GB,,2022/3/20,Emission intensity of stationary combustion of fuel. Fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Energy,Fuel,fuel_type_biopetrol-fuel_use_na,Bio Petrol,GJ,,,70.21,not-supplied,not-supplied,,,,Outside of scopes,fuel_combustion,BEIS,2021,2021,2021,GB,,2022/3/20,Emission intensity of stationary combustion of fuel. Fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1233]: All of the biofuel entries have these "outside of scope" entries - not sure what these are, could we have something in the name, description & ID to distinguish them from the other EFs?

File: OpenEmissionFactorsDB.csv [hunk 3]
```diff
@@ -1210,16 +1206,36 @@ Energy,Fuel,fuel_type_aviation_gasoline-fuel_use_aviation,Aviation gasoline,EUR,
...
-Energy,Fuel,fuel_type_avtur-fuel_use_aviation,Aviation turbine fuel (avtur),tonne,,3181.43,,,,,,,,fuel_combustion,BEIS,2021,,,GB,,2021/11/17,Emission intensity for aviation turbine fuel. The fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
+Energy,Fuel,fuel_type_avgas-fuel_use_aviation,Aviation spirit,kWh,,0.24375,0.23878,0.00269,0.00228,,,,1,fuel_combustion,BEIS,2021,2021,2021,GB,,2022/3/20,Emission intensity of stationary combustion of fuel. Fuels conversion factors should be used for primary fuel sources combusted at a site or in an asset owned or controlled by the reporting organisation. The emission factor is calculated using the Gross Calorific Value. Retrieved from the Conversion Factors 2021: flat file published by the UK BEIS/Defra at the source URL.,https://www.gov.uk/government/publications/greenhouse-gas-reporting-conversion-factors-2021
```
Comment (patstirling) [inline OpenEmissionFactorsDB.csv:1209]: All entries should have "not-supplied" in any of the gas columns that don't have a number

**Review by patstirling — CHANGES_REQUESTED** [review]

File: OpenEmissionFactorsDB.csv (hunk 3 above)
[the remark this answers is not in the document]
Comment (maryarbab) [reply OpenEmissionFactorsDB.csv:1209]: Would this break the existing IDs?

**Review by AnastasiaLobanova — CHANGES_REQUESTED** [review]
In principle I have only the question about the outside the scopes

**Review by AnastasiaLobanova — APPROVED** [review]

**Review by patstirling — CHANGES_REQUESTED** [review]

**Review by patstirling — APPROVED** [review]

**Pull request merged**